src/modules/productData.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const getData = async () => {
+  let headers = await sessionStorage.getItem("products");
```

Comment:
You don't need to pass in any headers. You are not authenticating a user.

Reply:
Okey, we will remove it and add it later on!